src/config/settings.ts

```diff
@@ -290,6 +290,26 @@ const settings: Settings = {
         ttl: parseInt(process.env.METRICS_REPORTING_LOCK_TTL || '60'),
       },
     },
+    liquidities: {
+      [ChainsIds.ETH]: {
+        [DexProtocolName.UNISWAP_V3]: {
+          interval: parseInt(process.env.UNISWAPV3_LIQUIDITY_JOB_INTERVAL || String(3 * getDayInMillisecond())),
+          lock: {
+            name: process.env.UNISWAPV3_LIQUIDITY_LOCK_NAME || 'lock::UniswapLiquidity',
+            ttl: parseInt(process.env.UNISWAPV3_LIQUIDITY_LOCK_TTL || '60'),
+          },
+        },
+      },
+      [ChainsIds.ARBITRUM]: {
```

Comment:
this is just for testing?

Reply:
Yeap, I want to deploy with multiple chains to see how scheduler and workers behave.